### PR TITLE
18fl final touches

### PR DESCRIPTION
**Pull request by ventusignis**
Fixed a very minor bug I found (erroneous browser console logging) and added the 70% ownership optional rule for 2p

### CODE REVIEW

File: lib/engine/game/g_18_fl/step/token.rb [hunk 1]
```diff
@@ -83,6 +83,7 @@ def token_cost_override(entity, city_hex, _slot, token)
             node = city_hex.respond_to?(:city?) ? city_hex : city_hex.tile.towns.first
```
Comment (tobymao) [inline lib/engine/game/g_18_fl/step/token.rb:83]: return unless (node = )

Comment (ventusignis) [reply lib/engine/game/g_18_fl/step/token.rb:83]: Wouldn't that be a syntax error? I could also do `return if !node`

Comment (tobymao) [reply lib/engine/game/g_18_fl/step/token.rb:83]: it is?

Comment (ventusignis) [reply lib/engine/game/g_18_fl/step/token.rb:83]: I've changed it to `return if node.nil?`

Comment (tobymao) [reply lib/engine/game/g_18_fl/step/token.rb:83]: return unless node

**Pull request merged**